### PR TITLE
Feature/asset information

### PR DESCRIPTION
**Pull request by SNuseibeh**

### CODE REVIEW

File: src/components/app/asset/asset.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  const stats = useMemo(() => {
+    return [
+      { title: t("mrkCap"), value: formatNumber(asset.mrkCap ?? 0) },
+      {
+        title: t("fullydiluted"),
+        value: formatNumber(asset.dilutedValuation ?? 0),
+      },
+      { title: t("circsupply"), value: asset.circlSupply },
+      { title: t("volume"), value: formatNumber(asset.volume ?? 0) },
+      { title: t("totalsupply"), value: asset.supply },
```
Comment (almujtabamohammed) [inline src/components/app/asset/asset.tsx:25]: Format Circulating Supply and Total Supply

File: src/components/shared/containers/asset/assetDetailsHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  const _assetprice = useSelector(selectAssetLivePrice);
+
+  return (
+    <Col className="gap-2">
+      <Row className="gap-2">
+        <ShadowButton
+          bgColor="bg-grey-5"
+          textColor="font-medium text-xs"
+          border="rounded"
+          onClick={() => console.log("d")}
```
Comment (almujtabamohammed) [inline src/components/shared/containers/asset/assetDetailsHeader.tsx:28]: remove console log

File: src/components/shared/containers/asset/assetDetailsHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          title={`${t("rank")} #${asset.rank ?? 0}`}
+        />
+        <ShadowButton
+          bgColor="bg-grey-5"
+          textColor="font-medium text-xs"
+          border="rounded"
+          onClick={() => console.log("d")}
```
Comment (almujtabamohammed) [inline src/components/shared/containers/asset/assetDetailsHeader.tsx:35]: Remove console log

File: src/components/shared/containers/asset/assetDetailsHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          iconSvg={<StarIcon className="w-4 h-4 fill-orage-1 stroke-0" />}
+          title="1.8k"
```
Comment (almujtabamohammed) [inline src/components/shared/containers/asset/assetDetailsHeader.tsx:37]: This 1.8k should not be static

File: src/components/shared/containers/asset/assetVote.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import {
+  ArrowTrendingUpIcon,
+  ArrowTrendingDownIcon,
+} from "@heroicons/react/24/outline";
+
+const AssetVote: FC = ({}) => {
+  return (
+    <Col className="gap-3">
+      <p className="font-medium text-sm">
+        How do you feel about Bitcoin today?
```
Comment (almujtabamohammed) [inline src/components/shared/containers/asset/assetVote.tsx:13]: Add translation

File: src/components/shared/containers/asset/assetVote.tsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        <ShadowButton
+          title="Bearish"
+          bgColor="bg-red-2"
+          onClick={() => null}
+          textColor="text-red-1"
+          border="rounded-md"
+          iconSvg={<ArrowTrendingDownIcon className="h-6 w-6 stroke-red-1" />}
+        />
+      </Row>
+      <p className="text-grey-1 text-xs">Vote to see results</p>
```
Comment (almujtabamohammed) [inline src/components/shared/containers/asset/assetVote.tsx:33]: Add translation

File: src/components/shared/containers/marketStats.tsx [hunk 7]
```diff
@@ -21,7 +21,7 @@ export const MarketStats: FC<MarketProps> = ({
         <p className="dark:text-[#F9FAFB] text-sm font-medium leading-5 items-center">
           {title}
         </p>
-        <AssetPnl value={value} textColor={textColor} isUp={+value > 0} />
+        <AssetPnl value={value + "%"} textColor={textColor} isUp={+value > 0} />
       </Row>
       <p className="dark:text-[#F9FAFB] text-xl font-medium leading-7 pt-2">
         $1,227,747,623,904
```
Comment (almujtabamohammed) [inline src/components/shared/containers/marketStats.tsx:27]: This value should not be static

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]

File: src/components/shared/containers/asset/assetDetailsHeader.tsx (hunk 4 above)
Comment (SNuseibeh) [reply src/components/shared/containers/asset/assetDetailsHeader.tsx:37]: for now its static until the backend returns this data

**Pull request merged**